protocol/avdl/stellar1/local.avdl

```diff
@@ -201,7 +201,8 @@ protocol local {
   // OutsideCurrencyCode examples: "USD", "EUR". Has to be one of
   // supported currencies, returned from getDisplayCurrenciesLocal.
   void changeDisplayCurrencyLocal(int sessionID, AccountID accountID, OutsideCurrencyCode currency);
-  CurrencyLocal getDisplayCurrencyLocal(int sessionID, AccountID accountID);
+  // Gets display currency for primary account if no accountID is given.
+  CurrencyLocal getDisplayCurrencyLocal(int sessionID, union { null, AccountID } accountID);
```

Comment:
@patrickxb OK with API change?

Reply:
It seems fine, but just so I know, what path in the frontend needs to call this when it doesn't have an account id?

Reply:
@patrickxb it's for prefilling the send form with the default display currency. When coming from chat / profile we haven't loaded any account IDs. When coming from the wallets tab we'll still pass in `accountID`

Reply:
Ok.  

Isn't GetWalletAccountsLocal() called pretty early on when the app starts up?  Then you would have the primary account ID.

Reply:
We did briefly call that on startup but we don't anymore